packages/vkui/src/components/UsersStack/UsersStack.tsx

```diff
@@ -140,7 +155,9 @@ export const UsersStack = ({
     const hrefID = `#${id}`;
     const maskID = `UsersStackMask${cmpId}${i}`;
 
-    return (
+    const photoSrc = typeof photo === 'object' ? photo.src : photo;
```

Comment:
чтобы два раза не проверять, можно вот так оставить

```suggestion
    const isPhotoType = typeof photo === 'object';
```

дальше уже можно вот такие условия писать

```ts
isPhotoType ? photo.src : photo;
```